### PR TITLE
Nn 2843 select location form

### PR DESCRIPTION
**Pull request by Nimphal**
Screenshot 2020-08-20 at 14 57 39

### CODE REVIEW

File: backend/controllers/cellMove/selectLocation.js [hunk 1]
```diff
@@ -1,18 +1,30 @@
...
   try {
     const prisonerDetails = await elite2Api.getDetails(res.locals, offenderNo, true)
     const nonAssociations = await elite2Api.getNonAssociations(res.locals, prisonerDetails.bookingId)
+    const cellAttributesData = await elite2Api.getCellAttributes(res.locals)
+    const locationsData = await whereaboutsApi.searchGroups(res.locals, prisonerDetails.agencyId)
+
+    const locations = locationsData.map(location => ({ text: location.name, value: location.key }))
+    locations.unshift({ text: 'All locations', value: 'ALL' })
+    const cellAttributes = cellAttributesData
+      .filter(cellAttribute => 'Y'.includes(cellAttribute.activeFlag))
```
Comment (jsrobertson) [inline backend/controllers/cellMove/selectLocation.js:26]: Could we just do `cellAttribute.activeFlag === 'Y'`?

**Review by jsrobertson — APPROVED** [review]
LGTM

File: backend/controllers/cellMove/selectLocation.js (hunk 1 above)
Comment (Nimphal) [reply backend/controllers/cellMove/selectLocation.js:26]: Yeah, I think this may be my Java side coming and Matt C advising that was preferable.

Comment (Nimphal) [reply backend/controllers/cellMove/selectLocation.js:26]: Will fix in the next ticket.

**Pull request merged**